R/methods-xcmsRaw.R

```diff
@@ -305,7 +305,8 @@ setMethod("findPeaks.matchedFilter", "xcmsRaw",
 setMethod("findPeaks.centWave", "xcmsRaw", function(object, ppm=25, peakwidth=c(20,50), snthresh=10,
                                                     prefilter=c(3,100), mzCenterFun="wMean", integrate=1, mzdiff=-0.001,
                                                     fitgauss=FALSE, scanrange= numeric(), noise=0, ## noise.local=TRUE,
-                                                    sleep=0, verbose.columns=FALSE, ROI.list=list()) {
+                                                    sleep=0, verbose.columns=FALSE, ROI.list=list(), 
+                                                    perform1stBaselineCheck=TRUE, roiScales=NULL) {
```

Comment:
Some comments here:
1) Is there a need to add these new parameters at all? They are not needed by your new `findPeaks` method.
2) Rename `perform1stBaselineCheck` to `firstBaselineCheck`.
3) You should perform some input argument checking, i.e. that length `roiScales` matches length of `ROI.list` and that `roiScales` contains numeric values.
4) Most importantly however: document the new arguments in the corresponding Rd file.